appserver/packager/legal/src/main/resources/glassfish/legal/3RD-PARTY-LICENSE-WEB-PROFILE.txt

```diff
@@ -1,4 +1,4 @@
-GlassFish Server 4.0 Web Profile THIRDPARTYLICENSEREADME.txt
+GlassFish Server 5.0 Web Profile THIRDPARTYLICENSEREADME.txt
```

Comment:
Please change this as:
GlassFish Server 5.0 Web Profile 3RD-PARTY-LICENSE-WEB-PROFILE.txt